README.md

```diff
@@ -25,7 +25,7 @@ most use-cases where you might want to reach for this package.
 
 ## Compatibility
 
-- Ember.js v3.20 or above
+- Ember.js v3.8 or above
```

Comment:
`ember-try` setup to run tests with scenarios back to 3.8, so technically we can claim 3.8